### PR TITLE
[Task][30142] Add Template Admin

### PR DESCRIPTION
**Pull request by thunv-1049**
https://edu-redmine.sun-asterisk.vn/issues/30142

### CODE REVIEW

**Comment by thunv-1049** [comment]
Screenshot from 2020-10-01 15-56-50

**Comment by thunv-1049** [comment]
ready

File: app/views/admin/_sidebar.html.erb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<aside class="main-sidebar sidebar-dark-primary elevation-4">
+  <%= link_to "#", class: "brand-link" do %>
+    <%= image_tag "logo.png", class: "brand-image img-circle elevation-3" %>
+    <span class="brand-text font-weight-light"><%= t ".admin" %> </span>
```
Comment (ghost) [inline app/views/admin/_sidebar.html.erb:4]:
```suggestion
    <span class="brand-text font-weight-light"><%= t ".admin" %></span>
```

File: app/views/layouts/admin.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <title><%= t ".title" %> </title>
```
Comment (ghost) [inline app/views/layouts/admin.html.erb:5]:
```suggestion
    <title><%= t ".title" %></title>
```

**Pull request merged**